src/plugins/data_source_management/public/components/data_source_view/data_source_view.tsx

```diff
@@ -7,15 +7,27 @@ import React from 'react';
 import { i18n } from '@osd/i18n';
 import { EuiPopover, EuiButtonEmpty, EuiButtonIcon, EuiContextMenu } from '@elastic/eui';
 import { SavedObjectsClientContract, ToastsStart } from 'opensearch-dashboards/public';
+import { IUiSettingsClient } from 'src/core/public';
 import { DataSourceOption } from '../data_source_menu/types';
-import { getDataSourceById } from '../utils';
+import { getDataSourceById, handleDataSourceFetchError } from '../utils';
 import { MenuPanelItem } from '../../types';
 
+export const LocalCluster: DataSourceOption = {
```

Comment:
can we extract it out since i think multiple places use this const

Reply:
okay